notification/manager.go

```diff
@@ -89,7 +89,9 @@ func (mgr *Manager) MessageStatus(ctx context.Context, messageID string, provide
 		trace.StringAttribute("provider.message.id", providerMsgID.ProviderName),
 	)
 	defer sp.End()
-	return checker.Status(ctx, providerMsgID.ExternalID)
+	status, err := checker.Status(ctx, providerMsgID.ExternalID)
+
```

Comment:
nit: move this line with the return
```suggestion

	status, err := checker.Status(ctx, providerMsgID.ExternalID)
```